.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,15 @@
+name: auto-merge
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
+        with:
+          github-token: ${{ secrets.QUIPPERTECH_GITHUB_API_TOKEN_FOR_PUBLIC_REPO }}
```

Comment:
I added this secret as an Organization Secret available to not only private repos but also public repeos.
![image](https://user-images.githubusercontent.com/241542/110602261-2854fc80-81c9-11eb-8dae-6aeced9e288f.png)

The scope of the token is `public_repo` only.
So I believe it may be safe even if it's leaked.
![image](https://user-images.githubusercontent.com/241542/110602366-4884bb80-81c9-11eb-8961-e228e45c0129.png)